Xamarin.PropertyEditing.Tests/MockObjectEditor.cs

```diff
@@ -69,7 +69,7 @@ public string TypeName
 
 		public event EventHandler<EditorPropertyChangedEventArgs> PropertyChanged;
 
-		public Func<IPropertyInfo, object, object> ValueEvaluator
+		public Func<IPropertyInfo, object, object, object> ValueEvaluator
```

Comment:
I think a doc comment is absolutely necessary here, as the object types obfuscate the semantics.

Reply:
It's only tests, but I can add something.